### PR TITLE
Add updated v0.6.0 stable PD driver specs - enable block tests for PD

### PR DESCRIPTION
**Pull request by davidz627**
/kind cleanup
/sig storage
/kind test
/assign @msau42 @mkimuram 
/priority important-soon

```release-note
NONE
```

### CODE REVIEW

**Comment by davidz627** [comment]
@msau42 tests are passing here now

File: test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml [hunk 1]
```diff
@@ -26,15 +25,23 @@ spec:
...
-          image: gcr.io/gke-release/csi-attacher:v1.2.0-gke.0
+          image: gcr.io/gke-release/csi-attacher:v2.0.0-gke.0
+          args:
+            - "--v=5"
+            - "--csi-address=/csi/csi.sock"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-resizer
+          image: gcr.io/gke-release/csi-resizer:v0.3.0-gke.0
```
Comment (msau42) [inline test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml]: enable the resizing tests too?

**Pull request merged**